src/mongo/Config.class.php

```diff
@@ -2026,6 +2064,48 @@ public function getTransactionLogDatabase($readPreference = \MongoClient::RP_PRI
     }
 
     /**
```

Comment:
These functions need not live in `Config`; they might be better placed in some kind of composite utilities class.  They're portable, though.